src/array_util.rb

```diff
@@ -0,0 +1,44 @@
+class ArrayUtil
+
+  def self.sum_even_numbers(numbers)
+    numbers.select(&:even?).sum
```

Comment:
You did not answer my question regarding nil guarding. 

A good approach is to init with a safe value if the received value is not great, you can do this with `||=` check out the docs and let's discuss this when we meet in person.

Reply:
I know about ```||```, but I didn't know about```||=```, nice.